### DISCUSSION TITLE
Increased tissue autofluorescence using LiBH4

### DISCUSSION THREAD

**Discussion by sfritzs** [IBEXImagingCommunity/ibex_imaging_knowledge_base#100] [Q&A]
Hi Community,

I am trying to implement multiplex imaging using IBEX in our lab and so far I am extremely happy with the fast and effective way of fluorophore bleaching. However, I have a problem with increasing tissue autofluorescence with each cycle of bleaching with LiBH4.
I did an experiment to compare hydrogenperoxide bleaching with LiBH4 bleaching. I had two adjacent human tonsil FFPE slices. I did an antibody staining at first which looked identical for both. Then I treated one slide with 4.5% H2O2 for 30min under white light and the other one with LiBH4 for 10min @ RT. The fluorophore bleaching was successful in both treatments and the tissue autofluorescence seemed to decrease during the H2O2 treatment. However, while the overall tissue autofluorescence decreased during LiBH4 treatment, the autofluorescence in the 488 channel for specific features (especially red blood cells and macrophages/eosinophils) increased dramatically with each bleaching. I performed another round of bleaching after that and again, the autofluorescence decreased further upon H2O2 treatment and increased upon LiBH4 treatment. Attached you can find example images and the mean+max bits for the ROI in the AF488 channel. 

I am wondering what I am doing wrong and hope you can help me. Looking forward to hear you opinions and thank you in advance for your help!

All the best,
Sonja
[Autofluorescence_example.pdf](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/files/13840392/Autofluorescence_example.pdf)

**Comment by radtkea** [comment]
Hi Sonja,

Thank you very much for your post and detailed results. I am happy that you have had great success with fluorophore bleaching, but I am sorry to hear that you are experiencing increased autofluorescence in the AF488 channel with red blood cells/macrophages/eosinophils. It's clear you are very careful. I don't think you are doing anything wrong.

The majority of IBEX users haven't experienced this issue or at least it wasn't a problem for their experiments. However, one of 
 my colleagues saw increased autofluorescence in the mouse spleen after eliminating signal with LiBH4. I am not sure why it would vary from person to person or tissue to tissue. 

I have a few other ideas/suggestions. Happy to discuss via Zoom, if you are interested. Feel free to email me (andrea.radtke@nih.gov).

Best wishes,
Andrea
